resources/js/components/menu.js

```diff
@@ -121,8 +141,8 @@ class MenuComponent extends LitElement {
                 <a href="/"><h4 slot="heading" class="menu-title">Go to
                   disciple.tools</h4></a>
                 ${this.menuItems.map(item => html`
-                  <a href="${item.href}">
-                    <sp-menu-item>${item.label}</sp-menu-item>
+                  <a href="${item.href}" class="menu-set">
```

Comment:
Please update the h4 at line 141 to be a sp-menu-item element, then match visually.